server/lib/PLS/Server/Request/TextDocument/PublishDiagnostics.pm

```diff
@@ -151,9 +151,16 @@ sub get_compilation_errors
     push @setup, (chdir => $new_cwd) if (length $new_cwd and -d $new_cwd);
 
     my @diagnostics;
+    my @loadfile;
```

Comment:
Can you make sure you are using these perltidy options?

-gnu -csc -l=200

Reply:
Good point...might need to also check in the .vscode directory to include workspace-specific PLS config.

I'll write an issue for that, but for now if you could tidy yourself.